index.js

```diff
@@ -1,17 +1,63 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item)
+    this.items.sort((a, b) => {
+      return a - b;
+    })
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < this.items.length) {
+      return this.items[pos]
+    } else {
+      throw new Error('Out of bounds')
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length > 0) {
+      return Math.max(...this.items)
+    } else {
+      throw new Error('EmptySortedList');
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.items.length > 0) {
+      return Math.min(...this.items)
+    } else {
+      throw new Error('EmptySortedList');
+    }
+  }
+  
 
-  sum() {}
+  sum() {
+    if (this.items.length > 0) {
+      return this.items.reduce((acc,next) => {
+        return acc + next;
+      })
+    } else {
+      return 0
+    }
+  }
 
-  avg() {}
+  avg() {
+    if (this.items.length > 0) {
+      const sum = this.items.reduce((acc,next) => {
+        return acc + next;
+      })
```

Comment:
puedes aprovechar el método this.sum() para evitar repetir la operación de suma ;)